yaml/oathkeeper/simple/oathkeeper-api.yaml

```
@@ -27,7 +30,7 @@ spec:
     spec:
       containers:
       - name: ory-oathkeeper
-        image: oryd/oathkeeper:v0.13.8_oryOS.8
+        image: oryd/oathkeeper:latest
```

While this makes sense from an outward perspective, this chart is really only being used by a tutorial we wrote. That tutorial specifically targets this version and is outdated for the actual latest versions. The same applies to the command etc. So this definitely needs to be reverted

```suggestion
        image: oryd/oathkeeper:v0.13.8_oryOS.8
```

---

I also noticed that that tutorial is pretty outdated. Isn't it an idea to write a new up-to-date one that reflects the current situation and also make a chart that is always up to date (and can be installed without helm)? I'd be willing to write that (mostly for the fact that I had quite a few troubles while deploying a istio-ambassador-ORY stack).

---

Absolutely! This has been on my agenda for so long but I simply lack time to do it. It would be awesome if you could help out and I'd be happy to help you along the way.

If you'd like, I can send over the markdown source file for the article?

---

Sure! I'm in the discord under the same username, or send through GitHub. And where do I send/merge it once it's finished?
I'll change this version back to v0.13.8 and make a new chart for the new tutorial.